android-interop-testing/build.gradle

```diff
@@ -3,9 +3,12 @@
 buildscript {
     repositories {
         jcenter()
+        mavenLocal()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.1.0'
+        // TODO(madongfly): Change to the released version once it is released.
+        classpath "com.google.protobuf:protobuf-gradle-plugin:0.5.0-SNAPSHOT"
```

Comment:
I have just pushed 0.5.0 to Maven Central. Please update the version and change mavenLocal() to mavenCentral() in the repositories block.